sygnal/gcmpushkin.py

```diff
@@ -152,13 +164,15 @@ async def create(cls, name, sygnal, config):
         """
         return cls(name, sygnal, config)
 
-    async def _perform_http_request(self, body, headers):
+    async def _perform_http_request(
+        self, body: Dict, headers: Dict[bytes, List[str]]
+    ) -> Optional[Tuple[IResponse, str]]:
```

Comment:
```suggestion
    ) -> Tuple[IResponse, str]:
```
I don't think this can return `None`?